src/components/LoginPage/Login/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useRef, useState } from "react";
+import { Form, Button, Card, Alert } from "react-bootstrap";
+import { Link, useHistory } from "react-router-dom";
+import { useAuth } from "../../contexts/AuthContext";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+export default function Login({ isTesting }) {
+  const { t } = useTranslation();
+  const emailRef = useRef();
+  const passwordRef = useRef();
+  const { login } = useAuth();
+  const [error, setError] = useState("");
+  const [loading, setLoading] = useState(false);
+  const history = useHistory();
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+
+    try {
+      setError("");
+      setLoading(true);
+      await login(emailRef.current.value, passwordRef.current.value);
+      history.push("/");
+    } catch {
+      setError("Failed to log in");
+    }
+
+    setLoading(false);
+  }
+
+  return (
+    <>
+      <Card className="shadow" style={{ marginTop: "10em" }}>
```

Comment:
classNames

Reply:
PTAL

Reply:
What does shadow mean, could it be more meaningful?

Reply:
that is bootstrap class, should I write it manually?

Reply:
No worries just add a comment to indicate that it’s a bootstrap class